docs/sources/features/datasources/cloudwatch.md

```diff
@@ -284,6 +336,6 @@ datasources:
       authType: keys
       defaultRegion: eu-west-2
     secureJsonData:
-      accessKey: "<your access key>"
```

Comment:
Noted that the quotes are changed here and I think that is valid YAML but worth checking to make sure.